### ISSUE TITLE
Minified all assets (gzip)

### ISSUE THREAD

**Issue by jankout** [codefog/contao-cookiebar#18]
Hello Kamil, 

it would be nice, when your extension could load all assets minified. Thank you.

**Comment by qzminski** [comment]
It does load them minified: https://github.com/codefog/contao-cookiebar/blob/master/classes/CookieBar.php#L30-L31

**Comment by jenda77** [comment]
I mean together with the another js files. It is possible with this example:

```
<?php
$GLOBALS['TL_JAVASCRIPT'][] = 
    'assets/jquery/colorbox/'.COLORBOX.'/js/colorbox.min.js|static';
?>
```

**Comment by qzminski** [comment]
It would be wrong because then you would have to download two big files instead of one:

1. One combined file with cookiebar
2. One combined file without cookiebar (once cookies are accepted)

**Comment by jenda77** [comment]
But it's important for Page Speed.

**Comment by CodeAlDente** [comment]
@qzminski I can't confirm that. If I take a look at my source code I can clearly see that cookiebar _is not loaded minified_ within the combined css/js file but instead Contao loads the files (for cookiebar) separately. What I want is a single file for css and js for better performance as pointed out by @jankout

**Comment by qzminski** [comment]
Implemented in b1dc548937419412207e02621daaca1633dc9d01.

**Comment by jankout** [comment]
thank you very much.
